src/weights.jl

```diff
@@ -28,6 +28,12 @@ isempty(wv::AbstractWeights) = isempty(wv.values)
 size(wv::AbstractWeights) = size(wv.values)
 Base.axes(wv::AbstractWeights) = Base.axes(wv.values)
 
+# https://github.com/JuliaLang/julia/pull/43354
+if VERSION >= v"1.8.0-DEV.1494" # 98e60ffb11ee431e462b092b48a31a1204bd263d
+    Base.allequal(wv::AbstractWeights) = allequal(wv.values)
+end
+Base.allunique(wv::AbstractWeights) = allunique(wv.values)
```

Comment:
I agree that `allequal` would be a good addition but I'm a bit ambivalent about `allunique`. Doesn't seem bad to have but I can't really think of a situation where it'd be useful. Do you have one in mind?

Reply:
No, I don't have a use-case in mind, I think it won't be used a lot. I mainly added it since it complements `allequal` (and while the fallback would be sufficient, as for `allequal`, one could potentially benefit if an optimied method exists for `weights.values`).